content/en/tracing/trace_collection/custom_instrumentation/cpp.md

```diff
@@ -54,13 +54,43 @@ To set tags across all your spans, set the `DD_TAGS` environment variable as a l
 
 ### Set errors on a span
 
-To customize an error associated with one of your spans, use the below:
+To associate a span with an error, set one or more error-related tags on the
+span. For example,
 
 ```cpp
 span->SetTag(opentracing::ext::error, true);
 ```
 
-Error metadata may be set as additional tags on the same span as well.
+Or, equivalently,
+
+```cpp
+span->SetTag("error", true);
+```
+
+Add more specific information about the error by setting any combination of the
+"error.msg", "error.stack", or "error.type" tags. See [Error Tracking][12] for
+more information about error tags.
+
+For example,
+
+```cpp
+// Associate this span with the "bad file descriptor" error from the standard
+// library.
+span->SetTag("error.msg", "[EBADF] invalid file");
+span->SetTag("error.type", "errno");
+```
+
+Setting any of "error.msg", "error.stack", or "error.type" has the effect of
```

Comment:
```suggestion
Adding any of the "error.msg", "error.stack", or "error.type" tags sets
```

Reply:
Perhaps these should be in back-tics as well.

Reply:
Ah, yes, good catch. I changed them to backticks.